docs_src/examples/Ch-ExamplesAddingSNMPDevice.md

```diff
@@ -44,7 +44,7 @@ Postman (Linux 64bit)
 EdgeX - barcelona-docker-compose.yaml
 
 \--
-<https://github.com/edgexfoundry/developer-scripts/blob/master/releases/barcelona/compose-files/docker-compose-barcelona-0.2.1.yml>
+<https://github.com/edgexfoundry/edgex-compose/blob/barcelona/docker-compose-barcelona-0.2.1.yml>
```

Comment:
Note that this link will be correct once PR for Barcelona branch is merged. Updating this example to Hanoi is out of scope.